src/stepper/src/components/styled/index.js

```diff
@@ -0,0 +1,76 @@
+import styled, { css } from 'styled-components';
+import { NEUTRALS } from '../../../../colors';
+
+const IconLayer = styled.span`
+  font-size: 24px;
+  margin-right: 8px;
+  min-width: 24px;
+`;
+
+const getDividerOrientationStyles = ({ vertical }) => {
+  if (vertical) {
+    return css`
+      min-height: 20px;
+      border-left: 2px solid ${NEUTRALS[4]};
+       /* step padding + half icon - 1 */
```

Comment:
Check indentation. There's an extra space before the comment block `/*`.

Reply:
![image](https://thumbs.gfycat.com/OrangeTightDipper-size_restricted.gif)